Vision/Vision.bs

```diff
@@ -107,76 +99,77 @@ Boilerplate: omit conformance
-		as defined in the [[ETHICAL-WEB-PRINCIPLES inline|Ethical Web Principles]].
 	* Our standards are rooted in a strong royalty-free patent policy
-		and open copyright licenses,
-		openly developed with consensus of industry and key stakeholders.
+		and open copyright licenses.
 	* We welcome individuals and organizations of all sizes
 		(from single-person companies to multi-nationals),
 		and take feedback from the general public.
-
-
-	# Principles and values # {#principles}
-
-	As W3C leads the Web forward,
-	our mission is to recognize and embody fundamental values and principles
-	into the architecture of the Web.
-	We must become more principled in our execution of the vision of the Web.
-	We must
-	* Ensure the Web is trustworthy,
-		by ensuring security and privacy for users.
-	* Aim to reduce centralization in Web architecture,
+	* We believe proven interoperable implementation is a 
+		requirement for broad adoption of standards. 
+		To ensure reliable interoperability, 
+		we require multiple implementations 
+		and open test suites for our standards.
+	* The Web will continue to expand 
+		in user base, global reach, and technical breadth. 
+		We are committed to encouraging incubation in new areas, 
+		collaborating on innovations across our community.
+	* We aim to reduce centralization in Web architecture,
 		minimizing single points of failure
 		and single points of control.
-	* Remain focused on ensuring reliable interoperability
-		through implementation experience
-		and open test suites.
-	* Implement a unified, extensible, Web architecture,
-		which continues to address evolving use cases for the general public.
-
-	We will do this by:
-	* Encouraging incubation in new areas and industries
-		with open platforms for discussion, collaboration, and innovation,
-		making it more structured
-		and improving consensus-building among key stakeholders.
-	* Striving for the broadest participation,
-		along axes including worldwide participation, diversity, and inclusion,
-		facilitating balance, equity, and cooperation
-		among the participants from different industries,
-		user groups, and organizational sizes,
-		and thus establishing W3C as representative of the whole community.
-	* Increasing involvement of under-represented key stakeholders
-		such as end users, content creators, and developers.
-	* Ensuring transparency, equity, and fairness.
-		Our work will not be exclusively dominated
-		by any person, company, or interest group.
-	* Ensuring the Web platform meets goals in horizontal principles such as
-		accessibility, internationalization, privacy, and sustainability.
-	* Establishing and improving collaborative relationships
-		with other organizations in the domain of Internet and Web standards,
+	* W3C is committed to establishing and improving collaborative relationships
+		with other Internet and Web standards organizations,
 		including building and maintaining respected relationships
```

Comment:
```suggestion
		and building and maintaining respected relationships
```
"including" feels off since governments and businesses aren't Internet and Web standards organizations.